[CS2113-T15-4] Diet Management Assistant

dejunnn opened the pull request:
@dejunnn 
@yuxianglim
@lamyuewei 
@DDzuikeai

alwayshuizhen commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,133 @@
 
+### Architecture
+
+![ArchitectureDiagram](/docs/images/Architecture.png)
```
The image is not rendered on the github pages

itskesin replied:
Same! I face similar issue! 😢  
I tried accessing the DG directly through github/docs/DeveloprGuide.md

alwayshuizhen commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,133 @@
+During the process a new`RecordMealCommand` object will be generated and returned to the `Logic` component.
+
+### Step1. Generate command
+![Step1. Diagram](/docs/images/Record_step1.png)
```
In lines 38 and 39 of this document, the `Logic` component consists of `DietManager-Arranges`. Perhaps it would be clearer if the diagram's object would be using the class you have come up with, `:Arranges`, instead of `:Logic`.

Here's a sample that you can consider:
![image](https://user-images.githubusercontent.com/32288711/77891100-0d80c500-72a3-11ea-991d-163b02bcfc3a.png)

alwayshuizhen commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,133 @@
+
+During execution, the command object will try to get a `record` object of the class `DailyFoodRecord`. The date of record specified by the user. 
+If no record of that day is found in profile, it will automatically generate a new record of that day. Then with `foodList` and `record`, can call 
+the method `record.recordMeals()`.
```
The use of `this with that can call func` might sound a little too informal and the calling of functions could be standardized by `class#method()`. Perhaps something like this will be better? 
> If no record of that day is found in profile, it will automatically generate a new record of that day. Together with `foodList` and `record`, a call will be made to `record#recordMeals()`.

Here's a sample from Week 10:
![image](https://user-images.githubusercontent.com/32288711/77891812-1faf3300-72a4-11ea-9e20-7dd2e8ba313a.png)

alwayshuizhen commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,133 @@
+![Step1. Diagram](/docs/images/Record_step1.png)
+
+Step 2.  The `Logic` composition calls the `execute()` method of the `RecordMealCommand` object. 
+
```
More diagrams could be helpful in explaining your subsequent steps 😄

alwayshuizhen commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,133 @@
+* `Food` - 
+* `FoodNutritionInfo` - 
+
+
```
More information would be helpful!

Janicetyy commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
 
+### Architecture
+
+![ArchitectureDiagram](/docs/images/Architecture.png)
```
Should the arrow from the Diet manager to the various components double-sided? Does the UI know about the DietManager?

itskesin commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+![ArchitectureDiagram](/docs/images/Architecture.png)
+
+The Architecture Diagram given above explains the high-level design of the Diet Manager Application.
+
```
The main component  `Main` is not elaborated at the start of the Architecture ? 🤔  It would be great to let the developers know ?

Janicetyy commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+* Reading user input and breaking it down to machine-readable code
+
+The Parser consists of the following classes: 
+* `Parser` - Breaks down inputs into machine-readable code
```
This might be a little confusing to me. The parser component reads input and breaks it down, but I had read that UI reads input? There is also only one class instead of more classes, or is this temporary?

Janicetyy commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+the method `record.recordMeals()`.
+
+### Step2. Execute and Save Result
+![Step2. Diagram](/docs/images/Record_step2.png)
```
Is it clearer if we avoid using double negation?

Janicetyy replied:
Is the execution of recordMealCommand happening at the same time as the function in logic that calls recordMealCommand#execute?

itskesin commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
-## Design & Implementation
+## Content
+1. [Design](#design)
+2. [Implementation](#implementation)
```
It would be great to include Sub-headings in the content page for readability! anyway, good job on the clickable content page 👍

Janicetyy commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+### Step2. Execute and Save Result
+![Step2. Diagram](/docs/images/Record_step2.png)
+
+#### 1.2 Design Considerations
```
Could it be better to include why one of the alternative choices was made?

Janicetyy commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+#### Aspect : Data structure to support the command
+* Alternative 1 (current choice): Use a list to store daily food record for a profile.
+    * Pros: Easy to implement and understand
+    * Cons: The list is maintained by a `Profile` object. Can lead to more duties for a `Profile` object.
```
And why more duties on Profile object is bad?

itskesin commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+The Food consists of the following classes: 
+* `Food` - 
+* `FoodNutritionInfo` - 
+
```
It would be great to include class diagram for individual components to ensure that all developers can understand the structure of the application clearly and see their relationship with each other better?  💭

Janicetyy commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+    * Pros: Easy to implement and understand
+    * Cons: The list is maintained by a `Profile` object. Can lead to more duties for a `Profile` object.
+* Alternative 2: Use a `FoodHistoryManager` to keep record of daily meals.
+    * Pros: More OOP.
+    * Cons: Makes the execution procedure more complex.
```
Why is OOP good? And how is the execution procedure made more complex?

itskesin commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+
+## Implementation
+
+{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
```
can replace this with general description on all the features?

Janicetyy commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+Given below is an example usage scenario and how the record mechanism behaves at each step.
+
+Step 1.  The `ui` object gets user input and sends it to the `parser`. The `parser` then parses the original input into a standard `command`
+and returns it. The command type is decided by the `commandPrompt`. The `RecordMealCommand` has `record-meal` as its commandPrompt. 
```
I didn't manage to find anything about the command prompt, maybe a little description somewhere would be useful?

itskesin commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+
+Given below is an example usage scenario and how the record mechanism behaves at each step.
+
+Step 1.  The `ui` object gets user input and sends it to the `parser`. The `parser` then parses the original input into a standard `command`
```
There's no mention of `Logic` object at the start of step 1 ? 
I guess it would be better to include it since Logic object is calling the method in `UI`?

Janicetyy commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+Step 2.  The `Logic` composition calls the `execute()` method of the `RecordMealCommand` object. 
+
+During the process of execution, the command object will generate a `ArrayList<Food> foodList`, which maintains all `Food` items listed in the user input. 
+If a certain kind of `Food` can be found in the database of `FoodNuritionInfo`, then this food item will be fetched from the database. 
```
How is the food item fetched from the database? There doesn't seem to be any mention of any database?

itskesin commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+During the process a new`RecordMealCommand` object will be generated and returned to the `Logic` component.
+
+### Step1. Generate command
+![Step1. Diagram](/docs/images/Record_step1.png)
```
I agree with @alwayshuizhen 's comment!

I believe that the sequence diagram should use the actual class name instead of  the general ` :Logic` component. In the description, you can states that `RecordMeal` command uses `Logic#exeecute()` to do_________ , to link the logic and the actual class name together 💪

itskesin replied:
For the sequence diagram, it could also be further split into different components with the background highlighted in different colors. ^ follow the example on addressbook-level3

itskesin replied:
For the sequence diagram, i guess it should include an arrow pointing to the :Logic to indicate the call to the start of the `:Logic` ? 🤔 
![image](https://user-images.githubusercontent.com/42372373/77988264-78d3a100-734e-11ea-8059-95c0eb2b95d2.png)

Janicetyy reviewed (commented):
Perhaps due to the time limitation, there were parts that still didn't make sense at the moment. There was a clear effort in explaining steps, but if they were shorter, perhaps it would be better to understand? Some descriptions of the methods used, or more diagrams to link the methods together could also help reader understand the process? Good work so far! :)

itskesin commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+* Allow users to track and monitor their eating habits
+* Provide personalised information and recommendations for the user
 
 ## User Stories
```
Do remember to include more user stories for each version 😄

itskesin commented on docs/DeveloperGuide.md:
```diff
@@ -1,18 +1,149 @@
+* Alternative 2: Use a `FoodHistoryManager` to keep record of daily meals.
+    * Pros: More OOP.
+    * Cons: Makes the execution procedure more complex.
+    
```
Overall, it would be nice to include more visuals e.g., screenshots/diagrams 💪

DDzuikeai committed:
Merge pull request #109 from DDzuikeai/master

CH-Ajust the "recommend" feature, to recommend recipe based on user's ph…

Commit:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

Commit:
DJ: Amended profile change commands

Commit:
DJ: Optimised Imports

Commit:
DJ: Fixed weight update duplicate command

Commit:
DJ: Amended Commands to prevent usage without existing profile

Commit:
DJ: Fixed checkstyle

Commit:
DJ: Fixed bug in setting profile

Commit:
DJ: Standardised UOM to centimetre for height

Commit:
DJ: Added a Result class to handle command results

dejunnn committed:
Merge pull request #110 from dejunnn/dejunnn-CodeArchitecture

Dejunnn code architecture

Commit:
DJ: Test

Commit:
Merge branch 'dejunnn-CodeArchitecture'

Commit:
DJ: Edited FoodNutritionRecord

Commit:
DJ: Minor edits

Commit:
DJ: Minor Edits

dejunnn committed:
Merge pull request #111 from dejunnn/master

DJ: Edited FoodNutritionRecord

yuxianglim1227 committed:
Added help function table

yuxianglim1227 committed:
Fixed Checkstyle

yuxianglim1227 committed:
deleted redundant files

yuxianglim1227 committed:
EXPECTED.txt

yuxianglim1227 committed:
EXPECTED.txt update

yuxianglim1227 committed:
More update

yuxianglim1227 committed:
zzz more edit

yuxianglim committed:
Merge pull request #112 from yuxianglim/master

Added help function table

Commit:
DJ: Reverted change to checkstyle

Commit:
DJ: Fixed checkstyle

dejunnn committed:
Merge pull request #113 from dejunnn/master

DJ: Reverted change to checkstyle

Commit:
DJ: Begun implementing Storage

Commit:
DJ: Implemented Storage function for Profile Information

Commit:
DJ: Implemented contingency when profile file is corrupted

lamyuewei committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

yuxianglim1227 committed:
Updated user guide

yuxianglim committed:
Merge pull request #115 from yuxianglim/master

Updated user guide

lamyuewei committed:
Update User Guide for check required calories

lamyuewei committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

# Conflicts:
#	docs/UserGuide.md

lamyuewei committed:
Merge pull request #116 from lamyuewei/master

Update user guide for check required calories

Commit:
DJ: Implemented Storage Function for Food Nutrtion Record

dejunnn committed:
Merge pull request #114 from dejunnn/dejunnn-Storage

DJ: Implementing Storage

lamyuewei committed:
Add J unit test for Food

lamyuewei committed:
Add line at end of file

lamyuewei committed:
Merge pull request #117 from lamyuewei/master

Add J unit test for Food

lamyuewei committed:
Edit Developer Guide for profile and food component

lamyuewei committed:
Resolve bugs from nus-pe-bot

lamyuewei committed:
Edit User Guide based on mock PE review

yuxianglim1227 committed:
Added Command Test

yuxianglim committed:
Merge pull request #122 from yuxianglim/master

Added Command Test

yuxianglim1227 committed:
added user-stories

yuxianglim committed:
Merge pull request #123 from yuxianglim/master

added user-stories to DG

yuxianglim committed:
Merge pull request #121 from lamyuewei/master

Edit UG, DG, fix bug

Commit:
DJ: Updated PPP

Commit:
DJ: Updated PPP

dejunnn committed:
Merge pull request #124 from dejunnn/master

DJ: Updated PPP

Commit:
DJ: Updated PPP

dejunnn committed:
Merge pull request #125 from dejunnn/master

DJ: Updated PPP

Commit:
DJ: Updated PPP

dejunnn committed:
Merge pull request #126 from dejunnn/master

DJ: Updated PPP

Commit:
DJ: Updated PPP

dejunnn committed:
Merge pull request #127 from dejunnn/master

DJ: Updated PPP

lamyuewei committed:
Add assertion

Commit:
DJ: Updated Directory management

dejunnn committed:
Merge pull request #128 from lamyuewei/master

Add assertion

Commit:
DJ: Resolve conflict

dejunnn committed:
Merge pull request #129 from dejunnn/master

DJ: Updated Directory management

Commit:
DJ: Abstracted app launcher

dejunnn committed:
Merge pull request #131 from dejunnn/master

DJ: Abstracted app launcher

lamyuewei committed:
Add PPP

yuxianglim committed:
Merge pull request #132 from lamyuewei/master

Add PPP

lamyuewei committed:
Add DG contribution for PPP

lamyuewei committed:
Updated AboutUs for dejun and me

yuxianglim1227 committed:
Added PPP

lamyuewei committed:
Merge pull request #133 from yuxianglim/master

Added PPP

Commit:
DJ: Minor changes to project structure

dejunnn committed:
Merge pull request #134 from dejunnn/master

DJ: Minor changes to project structure

lamyuewei committed:
Resolve bug of null string being stored

lamyuewei committed:
Merge branch 'yuewei-PPP'

lamyuewei committed:
Merge branch 'yuewei-AboutUs'

yuxianglim committed:
Merge pull request #135 from lamyuewei/master

Resolve bug for null string stored in record meal

yuxianglim1227 committed:
- Added track change to weight according to weight goal.
- Fixed bug on set-profile whereby past weight history is not reset
- Minor changes to PPP

yuxianglim1227 committed:
Fixed checkstyle spacing

yuxianglim1227 committed:
Updated test

yuxianglim1227 committed:
minor edit for checkstyle

lamyuewei committed:
Merge pull request #136 from yuxianglim/master

- Added ability to see progress to weight goal.

yuxianglim1227 committed:
Minor fix to spelling mistake
Removed unused string constant

lamyuewei committed:
Merge pull request #137 from yuxianglim/master

Minor fix to spelling mistake

Commit:
DJ: Minor changes

dejunnn committed:
Merge pull request #138 from dejunnn/master

DJ: Minor changes

yuxianglim1227 committed:
minor doc changes

Commit:
DJ: Updated coding standard and JavaDocs for Parsing Classes

lamyuewei committed:
Edit PPP for contirbution to team, review, and beyond

lamyuewei committed:
Merge branch 'yuewei-PPP'

yuxianglim1227 committed:
- Added check-bmi command
- BMI table

Commit:
DJ: Adjustments for command and Logic Management, updated Parsing coding standard

yuxianglim1227 committed:
Merge branch 'BMI-Branch'

Commit:
DJ: Optimisation and Reformatting

Commit:
DJ: JavaDocs for AppManager

Commit:
DJ: Minor coding standard edits

Commit:
DJ: Fixed checkstyle

dejunnn committed:
Merge pull request #139 from dejunnn/dejunnn-CodeStandard

DJ: Updated coding standard and JavaDocs

Commit:
Merge branch 'dejunnn-CodeStandard'

yuxianglim1227 committed:
- Show user's bmi value

yuxianglim1227 committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

 Conflicts:
	src/main/java/seedu/dietmanager/logic/parser/CommandParser.java
	src/main/java/seedu/dietmanager/logic/parser/WeightParser.java

yuxianglim1227 committed:
Updated Bmi command to parser

lamyuewei committed:
Merge pull request #140 from yuxianglim/master

Added BMI table and user's BMI

Commit:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

Commit:
DJ: Updated Storage function to support weight record

Commit:
DJ: Optimise imports and refromatting

Commit:
DJ: Restricted commands depending on profile validity

Commit:
DJ: Minor updates

lamyuewei committed:
Merge pull request #141 from dejunnn/master

DJ: Updated Storage

Commit:
DJ: Implemented JUnit testing

Commit:
DJ: Updated JUnit testing or Parsing

Commit:
DJ: Fixed Checkstyle

dejunnn committed:
Merge pull request #142 from dejunnn/dejunnn-Testing

DJ: Implemented JUnit testing

Chance-Yan committed:
Implement the storage of recipe

dejunnn committed:
Merge pull request #143 from DDzuikeai/master

CH-Implement the storage of recipe

Commit:
DJ: Update

Commit:
DJ: Minor Updates

Commit:
DJ: Minor Updates

dejunnn committed:
Merge pull request #144 from dejunnn/dejunnn-Testing

DJ: JUnit Testing

Chance-Yan committed:
Implement the storage of daily food record

dejunnn committed:
Merge pull request #145 from DDzuikeai/master

CH-Implement the storage of daily food record

Commit:
DJ: Updated Testing

Commit:
DJ: Fixed Checkstyle

dejunnn committed:
Merge pull request #146 from dejunnn/dejunnn-Testing

DJ: Updated Testing

Commit:
DJ: Update DG

dejunnn committed:
Merge pull request #147 from dejunnn/dejunnn-Testing

DJ: Update DG

Commit:
DJ: Updated packages

Commit:
DJ: Update DG

Chance-Yan committed:
Remove some magic strings and implement the feature to clear all records

Commit:
DJ: Update DG

dejunnn committed:
Merge pull request #148 from dejunnn/dejunnn-Testing

Dejunnn testing

Commit:
DJ: Updated documentation

Chance-Yan committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

# Conflicts:
#	src/main/java/seedu/dietmanager/logic/parser/CommandParser.java

dejunnn committed:
Merge pull request #150 from dejunnn/dejunnn-Documentation

DJ: Updated documentation

dejunnn committed:
Merge pull request #149 from DDzuikeai/master

CH-Remove some magic strings and implement the feature to clear all records

Commit:
DJ: Update AboutUs

Commit:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp into dejunnn-Documentation

Commit:
DJ: Updated UG

Commit:
DJ: Update UG

Commit:
DJ: Update UG - Profile Commands almost done

Commit:
DJ: Fixed formatting

dejunnn committed:
Merge pull request #151 from dejunnn/dejunnn-Documentation

Dejunnn documentation

Chance-Yan committed:
Update CH's PPP

Chance-Yan committed:
Update Chenghao's PPP

DDzuikeai committed:
Set theme jekyll-theme-cayman

DDzuikeai committed:
Merge pull request #152 from DDzuikeai/master

Update CH's PPP

Commit:
DJ: Updated UG

dejunnn committed:
Merge pull request #153 from dejunnn/dejunnn-Documentation

DJ: Updated UG

Chance-Yan committed:
Fix a bug

Chance-Yan committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

DDzuikeai committed:
Merge pull request #154 from DDzuikeai/master

Fix a bug related to recipe

Commit:
DJ: Updated UG

Commit:
DJ: Update text-ui-testing

Commit:
DJ: Updated text-ui.testing

Commit:
DJ: updated text-ui

dejunnn committed:
Merge pull request #155 from dejunnn/dejunnn-Documentation

DJ: Updated UG

Commit:
DJ: Text-ui-testing

Commit:
DJ: Updated text-ui-testing

Commit:
DJ: Remove data file

dejunnn committed:
Merge pull request #156 from dejunnn/dejunnn-Documentation

DJ: Text-ui-testing

lamyuewei committed:
Update PPP

lamyuewei committed:
Update PPP (delete optional portion)

lamyuewei committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

lamyuewei committed:
Merge pull request #157 from lamyuewei/yuewei-PPP

Yuewei ppp

lamyuewei committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

lamyuewei committed:
Edit DG

yuxianglim committed:
Merge pull request #158 from lamyuewei/yuewei-DG

Edit DG

yuxianglim1227 committed:
Edited PPP and DG

lamyuewei committed:
Merge pull request #159 from yuxianglim/PPP-branch

Edited PPP and DG

yuxianglim1227 committed:
- Typo error edit

lamyuewei committed:
Merge pull request #160 from yuxianglim/PPP-branch

- Typo error edit

lamyuewei committed:
Update PPP

lamyuewei committed:
Merge branch 'master' into yuewei-PPP

Commit:
DJ: Updated UG and DG

Commit:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp into dejunnn-Documentation

 Conflicts:
	docs/DeveloperGuide.md

yuxianglim1227 committed:
Updated and reorganised help table

lamyuewei committed:
Merge pull request #161 from dejunnn/dejunnn-Documentation

Dejunnn documentation

lamyuewei committed:
Updated user guide

lamyuewei committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

yuxianglim1227 committed:
changed expected.txt

lamyuewei committed:
Edit DG to add YX's edit

Commit:
DJ: Updated DG

lamyuewei committed:
Merge pull request #162 from yuxianglim/PPP-branch

Updated and reorganised help table

dejunnn committed:
Merge pull request #163 from dejunnn/dejunnn-Documentation

DJ: Updated DG

Commit:
DJ: Updated DG

Commit:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp into dejunnn-Documentation

lamyuewei committed:
Merge branch 'yuewei-PPP'

dejunnn committed:
Merge pull request #164 from dejunnn/dejunnn-Documentation

Dejunnn documentation

lamyuewei committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

lamyuewei committed:
Edit DG

yuxianglim1227 committed:
Updated UG

Chance-Yan committed:
Update ddzuikeai.md

lamyuewei committed:
Merge pull request #165 from yuxianglim/master

Updated UG

lamyuewei committed:
Update DG (ABC Parser)

lamyuewei committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

lamyuewei committed:
Edit DG (ABC Command)

lamyuewei committed:
Edit DG (Exception)

lamyuewei committed:
Update PPP

lamyuewei committed:
Edit UG

lamyuewei committed:
Edit UG

dejunnn committed:
Merge pull request #166 from lamyuewei/master

Edit DG for ABC parser, Command, Exceptions

Commit:
DJ: Updated DG

Commit:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp into dejunnn-Documentation

Commit:
DJ: Updated DG and Text-ui-testing

dejunnn committed:
Merge pull request #167 from dejunnn/dejunnn-Documentation

Dejunnn documentation

Commit:
DJ: Update PPP

Commit:
DJ: Minor updates

dejunnn committed:
Merge pull request #168 from dejunnn/dejunnn-Documentation

DJ: Update PPP

Commit:
DJ: Update PPP

dejunnn committed:
Merge pull request #169 from dejunnn/dejunnn-Documentation

DJ: Update PPP

yuxianglim1227 committed:
Minor update to help table

yuxianglim1227 committed:
Minor changes to help table

yuxianglim1227 committed:
EXPECTED.txt changes

Chance-Yan committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

yuxianglim1227 committed:
minor changes

yuxianglim committed:
Merge pull request #171 from yuxianglim/master

Minor changes to help table and Expected.txt

Chance-Yan committed:
Update Chenghao's PPP file

yuxianglim1227 committed:
updated ppp

Chance-Yan committed:
Update ddzuikeai.md

yuxianglim1227 committed:
Updated PPP
Removed unused code in DailyFoodRecord

DDzuikeai committed:
Merge pull request #173 from DDzuikeai/master

CH-Update PPP file

dejunnn committed:
Merge pull request #172 from yuxianglim/master

updated ppp

yuxianglim1227 committed:
updated aboutus

yuxianglim1227 committed:
Updated About Us

yuxianglim1227 committed:
Updated picture

yuxianglim committed:
Merge pull request #174 from yuxianglim/master

updated aboutus

lamyuewei committed:
Add Junit testing for commands

lamyuewei committed:
Fix PR checks

dejunnn committed:
Merge pull request #175 from lamyuewei/master

Add Junit testing for commands

lamyuewei committed:
Fix PR checks

Commit:
DJ: Update PPP

Commit:
DJ: Final Commit before release

dejunnn committed:
Merge pull request #176 from dejunnn/dejunnn-Documentation

DJ: Final PR

Commit:
Merge branch 'dejunnn-Documentation'

dejunnn committed:
Merge pull request #177 from dejunnn/master

Final Push

lamyuewei committed:
Edit PPP

lamyuewei committed:
Merge pull request #178 from lamyuewei/master

PPP update

lamyuewei committed:
Edit PPP hyperlink

lamyuewei committed:
Merge pull request #179 from lamyuewei/master

Edit PPP hyperlink

yuxianglim1227 committed:
edited ppp link

yuxianglim1227 committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

yuxianglim committed:
Merge pull request #180 from yuxianglim/master

edited ppp link

yuxianglim1227 committed:
updated yuewei's picture

lamyuewei committed:
Add photo

yuxianglim1227 committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

lamyuewei committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T15-4/tp

# Conflicts:
#	docs/AboutUs.md

lamyuewei committed:
Merge pull request #181 from lamyuewei/master

Add photo

yuxianglim committed:
Merge branch 'master' into master

yuxianglim committed:
Merge pull request #182 from yuxianglim/master

updated yuewei picture
